cmd/deploy.go

```diff
@@ -297,37 +297,47 @@ func installHelmRelease(ctx context.Context, helmClient *helm.Client, releaseNam
 	spinner.StopFailMessage("groundcover helm release installation failed")
 	defer spinner.Stop()
 
-	var release *helm.Release
 	helmUpgradeFunc := func() error {
-		if release, err = helmClient.Upgrade(ctx, releaseName, chart, chartValues); err != nil {
+		if _, err = helmClient.Upgrade(ctx, releaseName, chart, chartValues); err != nil {
 			return ui.RetryableError(err)
 		}
 
 		return nil
 	}
 
-	if err = spinner.Poll(ctx, helmUpgradeFunc, HELM_DEPLOY_POLLING_INTERVAL, HELM_DEPLOY_POLLING_TIMEOUT, HELM_DEPLOY_POLLING_RETIRES); err != nil {
+	err = spinner.Poll(ctx, helmUpgradeFunc, HELM_DEPLOY_POLLING_INTERVAL, HELM_DEPLOY_POLLING_TIMEOUT, HELM_DEPLOY_POLLING_RETIRES)
+
+	if err == nil {
+		return nil
+	}
+
+	if errors.Is(err, ui.ErrSpinnerTimeout) {
+		sentry_utils.SetLevelOnCurrentScope(sentry.LevelWarning)
+		spinner.SetWarningSign()
+		spinner.StopFailMessage("Timeout waiting for helm release installation")
```

Comment:
we should add some text about, retry or wait

Reply:
when the helm install is timeout, the next step is the validation process